### ISSUE TITLE
Type exists in two different assemblies build error in fresh project

### ISSUE THREAD

**Issue by SamWilliamsGS** [MindscapeHQ/raygun4net#492]
**Expected**
The type `RaygunSendingMessageEventArgs` can be used when you have the following combination of packages installed:
- `Mindscape.Raygun4Net.AspNetCore` version `6.7.0`
- `Mindscape.Raygun4Net.NetCore.Common.Signed` version `6.7.0`
- `Mindscape.Raygun4Net.NetCore.Signed` version `6.7.0`

**Actual**
Build fails with the following error:
`Error CS0433 : The type 'RaygunSendingMessageEventArgs' exists in both 'Mindscape.Raygun4Net.NetCore.Common, Version=6.7.0.0, Culture=neutral, PublicKeyToken=e1f29eaff63103db' and 'Mindscape.Raygun4Net.NetCore.Common, Version=6.7.0.0, Culture=neutral, PublicKeyToken=null'`

**Steps to Reproduce**
- Create a new ASP.NET web project. I used the default weather forecast one from JetBrains Rider.
- Install the three nuget packages mentioned above.
- Create a function taking `RaygunSendingMessageEventArgs` as an argument and `import Mindscape.Raygun4Net` as usual.
- Try to build the project.

**Potential Leads**
It's possible given the conflicting package versions have differing public key tokens, that there's an issue with the signing. A keen-eyed colleague noticed that versions [6.5.0](https://nuget.info/packages/Mindscape.Raygun4Net.NetCore.Common/6.5.0) and [6.7.0](https://nuget.info/packages/Mindscape.Raygun4Net.NetCore.Common/6.7.0) of the implicitly referenced `Mindscape.Raygun4Net.NetCore.Common` package are signed by different entities. 6.5.0 is signed by "Symantec SHA256 TimeStamping Signer" and 6.7.0 is signed by "DigiCert Timestamp 2023".

YMMV though, this could be a red herring. 🐟

**Comment by phillip-haydon** [comment]
Hello Sam,

Thank you for raising this issue, I have published 6.7.1 for the AspNetCore package.

For some reason when building the package it targeted `Mindscape.Raygun4Net.NetCore.Common` instead of `Mindscape.Raygun4Net.NetCore.Common.Signed`

Let me know if you have any other issues.

**Comment by phillip-haydon** [comment]
Hello @erwan-joly 

6.6.6 has a dependency on `Mindscape.Raygun4Net.NetCore.Common.Signed`, and 6.7.0 I accidently built with a dependency on `Mindscape.Raygun4Net.NetCore.Common` (the one without .Signed suffix)

If you included `Mindscape.Raygun4Net.NetCore.Common` then it would cause a conflict between the signed and unsigned version. 

You should remove the package without .Signed and include the package with .Signed. i.e

Remove: Mindscape.Raygun4Net.NetCore.Common
Add: Mindscape.Raygun4Net.NetCore.Common.Signed

(you may need to clear out the obj/bin directories, though just testing this now it seemed to resolve them correctly in a .NET 7 project going from 6.6.6 to 6.7.0 then 6.7.1)

**Comment by erwan-joly** [comment]
@phillip-haydon yeah I tried this but it does not works with the code that use ApplicationVersion and UserInfo

example that does not compile with Mindscape.Raygun4Net.NetCore.Common.Signed

public class RaygunClientProvider : DefaultRaygunAspNetCoreClientProvider
{
    public override RaygunClient GetClient(RaygunSettings settings)
    {
        var client = base.GetClient(settings);
         client.ApplicationVersion = version;
        return client;
    }
}

**Comment by phillip-haydon** [comment]
@erwan-joly I've published 6.7.2 for the aspnetcore + signed packages.

I took your example, made sure it compiled and worked on 6.6.6, then updated to 6.7.1 which broke, then to 6.7.2 which works. 

Please do let me know if it works (or doesn't work) for you. 

Appreciate the feedback!

**Comment by erwan-joly** [comment]
> @erwan-joly I've published 6.7.2 for the aspnetcore + signed packages.
> 
> I took your example, made sure it compiled and worked on 6.6.6, then updated to 6.7.1 which broke, then to 6.7.2 which works.
> 
> Please do let me know if it works (or doesn't work) for you.
> 
> Appreciate the feedback!

I can confirm it’s fixed for me 🙂

**Comment by SamWilliamsGS** [comment]
Fixed for me as well. Thanks for the quick response to this 🙂
